### DISCUSSION TITLE
Error message should be reworked

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#110570] [Mobile] [answered]
GitHub Mobile beta for Android

![Screenshot_20240229-200336.png](https://github.com/community/community/assets/2119212/c8e6ca45-9365-4f44-b3b6-ce16bc7bcd9d)

> Could not resolve to an issue or pull request with the number of 67.
>
> TRY AGAIN

The `with the number of` is incredibly awkward.

**Comment by stevepopovich** [comment]
Hi @jsoref what suggestion do you have to replace this text with?

**Reply by jsoref** [reply, marked as answer]
Hmmm. Possibly:

```diff
-Could not resolve to an issue or pull request with the number of 67.
+Could not find a pull request or issue numbered 67.
```

(It isn't great, but I think it's better than the current text.)


Notes:
- `resolve` is an unnecessarily technical term -- replacing it with _find_ feels like a definite improvement
- I can't quite explain why I reordered pull request before issue -- in this case, in 99% certain the URL was a /pull/
- replacing `with the number of` with _numbered_ feels much better

In this case, the repository has 0 issues/pull requests.

Note that "Try again" is unlikely to work in such cases.

Having easy access to the URL in question would be more useful in such cases than a try again button.

**Reply by stevepopovich** [reply]
Cool, thanks for the feedback!
